CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+#Contributing
+
+Thank you for your interest in contributing to Eventbrite's Javascript SDK!
+
+###Table of Contents
+* [Workflow](#workflow)
+  * [Setup](#setup)
+  * [Branches](#using-branches-to-submit-changes)
+  * [Keeping up to date](#keeping-your-local-repo-up-to-date)
+* [Creating issues](#creating-issues)
+* [Working on and submitting changes](#working-on-and-submitting-changes)
+  * [Steps to submit](#steps-to-submit)
+
+## Workflow
+
+### Setup
+
+1. Fork the repository
+2. Clone your new forked repository to your local computer
+3. Set the Eventbrite repository as your branches upstream branch
+`git remote add upstream https://github.com/eventbrite/eventbrite-sdk-javascript.git`
+4. Navigate to the root directory of your newly cloned repository 
+(for mac `cd /path/to/eventbrite-sdk-javascript`)
+5. `yarn install` to install local dependencies
+
+### Using branches to submit changes
+To work on changes to the Eventbrite repository, create a new branch on your local repository. `git checkout -b <your-new-branch-name>`
+
+
+### Keeping your local repo up to date
+To ensure your branch never gets out of sync with Eventbrite's master, ensure that you have your upstream set properly (see the [Setup](#setup) step)
+
+1. `git checkout master` (you may have to stash or commit your local changes if on a new branch)
+2. `git pull upstream master`
+3. `git checkout <your-new-branch-name>`
+4. `git rebase master`
+5. If you've stashed changes, unstash them now, otherwise your branch should now be up to date
+
+Always try to keep your master 'clean' by only pulling changes directly from upstream into your master branch and rebasing those changes onto your working branch.
+
+It is always a good idea to pull the upstream branch in to your master branch before creating a new feature branch to work from. This will minimize the chances of encountering merge conflicts.
+
+## Creating Issues
+Create issues to File bugs, changes, and proposals.
+
+Before opening a new issue, please [search][issues] to see if there has been previous discussion about the same feature or issue. If so, please contribute to the discussion there. 
+
+If nothing is found, feel free to [open a new issue][issues] and fill out the issue template to the best of your ability.
+
+## Working on and submitting changes
+When starting on improvements or new features that are non-trivial, it is always a good idea to first discuss the changes you wish to implement by [opening a github issue][issues] before getting started. 
+
+If you've found a bug or feature you'd like to work on in our [github issue tracker][issues], please comment on the issue to let others know that you'd like to work on it.
+
+While implementing fixes, please try to change as little code as possible. This helps speed up the review process and helps diminish the chance of additional bugs. 
+
+Please try to conform to the coding style of the code base.
```

Comment:
Not sure how specific we want to be put, but either a link to the coding style or how they could verify that. ie `yarn run lint && yarn run format`

Reply:
Code gets automatically formatted on commit, so they should be good there. And then linting runs in Travis as part of `validate` script. I actually don't think we're running prettier verification though. If someone bypasses prettier when committing they could actually put a PR up w/ unformatted code I believe...

Reply:
I think much like validating the title, we can probably find something to validate the the format is proper. Something like `eslint-config-prettier` should do the job.